full_snapshot.proto

```diff
@@ -458,6 +477,9 @@ message RelationStatistic {
   int64 tidx_blks_hit = 25;       // Number of buffer hits in this table's TOAST table indexes (if any)
   int64 toast_size_bytes = 26;    // TOAST table and TOAST index size (included in size_bytes as well)
   NullTimestamp analyzed_at = 27;
+
+  uint32 frozen_xid_age = 30; // The frozen xact ID age for this relation (relative to latest stable xid)
+  uint32 min_mxid_age = 31; // The minimum multixact ID age for this relation (relative to latest stable mxid)
```

Comment:
Nitpicking, but I wonder if `oldest_mxid_age` (i.e. the age of the oldest mxid) would be a better way to describe this - to quote the Postgres docs:

> All multixact IDs before this one have been replaced by a transaction ID in this table.

So its the oldest multixact ID that is still present. If we're talking about this in the context of age, I think it could also be considered the "maximum mxid age" (across all tuples), but I don't think "minimum" is correct here, unless I'm missing something.

Reply:
> Uhmm, I think you're putting minimum on the "age", not "mxid"?

Yeah, exactly, the way I read the name is **minimum** of **mxid age**, not **age** of **minimum mxid** (which is how I believe you're interpreting it).

To me "oldest" does not have that problem, since it can be read either way, i.e. **oldest** of **mxid age** and **age** of **oldest mxid** means the same thing.

But no strong opinion, it just jumped out to me when I read over it.

Maybe @msakrejda has a thought on the naming here?

Reply:
Uhmm, we could change it from `min_mxid_age` to `minmxid_age` (in that case, maaaaybe we should also do `frozenxid_age`), how about that?

Reply:
Sure, that would work. This is also easy to change (protobuf names don't need to match between parties, its only the number that matters), so no need to hold up the PR for this :)

Reply:
I'll go with `minmxid_age` and `frozenxid_age`, also will tweak the comment from "The minimum multixact ID age for this relation" to "The age of minimum multixact ID for this relation" 👍